modules/mod_navigation.js

```diff
@@ -0,0 +1,15 @@
+export default class Navigation {
+  constructor(pages) {
+    this.pages = pages;
+  }
+
+  show(pageIdToShow, display) {
+    this.pages.forEach((page) => {
+      if (pageIdToShow === page.id) {
+        page.style.display = display;
+      } else {
+        page.style.display = 'none';
+      }
+    });
+  }
```

Comment:
- [ ] It will be great to make this an arrow function. Remember we have to ensure our code follows the new es6 syntax. 
**HINT**
```
show = (pageIdToShow, display) => {
yourcode 
}
```